src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,6 +30,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        String []info = address.split(", ");
+        this.block.addInfo(info[0]);
+        this.street.addInfo(info[1]);
+        this.unit.addInfo(info[2]);
+        //this.postalCode.addInfo(Integer.parseInt(info[3]));
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.